archivo/style.css

```diff
@@ -5,25 +5,16 @@ Author: Automattic
 Author URI: https://automattic.com/
 Description: Archivo is a blog and portfolio theme that shows your featured image large.
 Requires at least: 5.8
```

Comment:
Because it uses `css` in theme.json, update min version to 6.3

Reply:
Oh, thank you! That's good to know. I'll change that now.